docker/build-sudo-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Build fast OS docker images that run slower steps ahead of time, to save time running tests
+
+cd "$(dirname "$0")"
+sed -i .orig -e "s#@@MYUSERID@@#`id -u`#g" sudo-*/Dockerfile
+docker build -t sudo-debian:wheezy sudo-debian
+docker build -t sudo-centos:6 sudo-centos6
+docker build -t sudo-centos:7 sudo-centos7
+docker build -t sudo-opensuse:13.2 sudo-opensuse
+docker build -t sudo-ubuntu:14.04 sudo-ubuntu14
```

Comment:
@rsandell Are you suggesting 12.04 should be one of the test images here?  It's fairly easy to add a new image (does take a bit of tweaking to ensure it behaves correctly though).   I suppose the question is: what's the smallest set of base images we can test against to give reasonable coverage of service behaviors?